docs/contributing/detailed-setup-guide.md

```diff
@@ -4,7 +4,7 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 <!-- Automatically created with yarn run createtoc and on push hook -->
 
-- [Chat](#chat)
+- [High level diagram](#high-level-diagram)
```

Comment:
Thanks!

Reply:
You're welcome. Although to be perfectly honest I have no idea how this appeared here

Reply:
I'm guessing this: `Automatically created with yarn run createtoc` did it

Reply:
Yup - may be that @shtukas's hooks are not running correctly?